jwst/stpipe/core.py

```diff
@@ -80,6 +80,20 @@ def _get_crds_parameters(cls, dataset):
         # for all other cases, use read_metadata directly to lazy-load
         return (read_metadata(dataset, flatten=True), crds_observatory)
 
+    @staticmethod
+    def get_known_loggers():
+        """
+        Get the names of loggers to configure.
+
+        Returns
+        -------
+        loggers : list
+            List of log names to configure.
+        """
+        # Specify the log names for any dependencies whose
+        # loggers we want to configure
+        return ["stpipe", "jwst", "stcal", "stdatamodels"]
```

Comment:
Should this be immutable? Also should it be ordered alphabetically? I don't see this used anywhere in this PR, so is this a building block for a future PR?
```suggestion
        return ("stpipe", "jwst", "stcal", "stdatamodels")
```

Thanks!

Reply:
Order would reduce the risk of humans from accidentally entering duplicate names, unless you want to make it a `set()` instead.